tests/Util/isAllowedDomain.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+Util: test isAllowedDomain()
+--FILE--
+<?php
+
+require_once __DIR__ . '/../loadConfig.php';
+
+$allowed_domains = array(
+    'test.com',
+    'test.edu'
+);
+
+$boolarray = Array(false => 'false', true => 'true');
+
+echo $boolarray[\UNL\VisitorChat\Util::isAllowedDomain('http://test.com', $allowed_domains)] . PHP_EOL;
+echo $boolarray[\UNL\VisitorChat\Util::isAllowedDomain('https://test.com', $allowed_domains)] . PHP_EOL;
+echo $boolarray[\UNL\VisitorChat\Util::isAllowedDomain('http://test.com/stuff.php?test', $allowed_domains)] . PHP_EOL;
+echo $boolarray[\UNL\VisitorChat\Util::isAllowedDomain('http://bad.com', $allowed_domains)] . PHP_EOL;
+echo $boolarray[\UNL\VisitorChat\Util::isAllowedDomain('http://bad.com/stuff.php?test', $allowed_domains)] . PHP_EOL;
+echo $boolarray[\UNL\VisitorChat\Util::isAllowedDomain('http://bad.com/stuff.php?test=http://test.com', $allowed_domains)] . PHP_EOL;
+echo $boolarray[\UNL\VisitorChat\Util::isAllowedDomain('http://subdomain.test.com', $allowed_domains)] . PHP_EOL;
```

Comment:
Beautiful tests. Add the following should be false:

* `test.com.bad.com`
* `iamnottest.edu`